### PR TITLE
[feat] Scenario Overview

### PR DESCRIPTION
**Pull request by itsliterallymonique**
### 🦋 Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### 🦧 Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## 🐸 How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## 🐙 Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### 🐁 Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @me-liu

### CODE REVIEW

File: app/participants/scenario-overview/page.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+export default function ScenarioOverview() {
+  const { sessionId } = useParams<{ sessionId: string }>();
+
+  //const [templateId, setTemplateId] = useState< UUID | null>(null);
+  const [templateInfo, setTemplateInfo] = useState<Template | null>(null);
+  //const router = useRouter();
+
+  const loadData = useCallback(async () => {
+    if (!sessionId) return;
+    let templateId = await fetchTemplateId(sessionId as UUID);
```
Comment (eshabansiya) [inline app/participants/scenario-overview/page.tsx]: 'templateId' is never reassigned. Use 'const' instead

File: app/participants/scenario-overview/page.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          <ContentHeader>Setting</ContentHeader>
+          <ContentBody>
+            {templateInfo ? templateInfo.setting : "no setting"}
+          </ContentBody>
+        </ContentBubble>
+        <ContinueButtonDiv>
+          <ContinueButton>
+            <ButtonText>Continue</ButtonText>
+          </ContinueButton>
+        </ContinueButtonDiv>
```
Comment (eshabansiya) [inline app/participants/scenario-overview/page.tsx]: please removed the nested headers - ContentDiv (just put padding/width on Main), and ContinueButtonDiv (just use align-self: flex-end on the button), and ButtonText (just put the text styles directly on ContinueButton)
